src/libvirtApi/domain.js

```diff
@@ -588,6 +588,7 @@ export function domainInstall({ onAddErrorNotification, vm }) {
     return cockpit
             .spawn([pythonPath, "--", "-", args], opts)
             .input(installVmScript)
+            .fail(ex => console.error(JSON.stringify(ex)))
```

Comment:
.catch() please, let's avoid introducing jQuery promise API.

Reply:
catch and fail are slightly different, in that catch will return a new (resolved) promise, whereas fail will return the original promise

In this case I want to propagate the original failure, for the UI error handling to show the error as well.

Reply:
I think I can use then instead.

Reply:
Interesting, I was not aware of that. Thanks!